main.ts

```diff
@@ -1,30 +1,30 @@
 class Student {
-    Id:number;
-    Name:string;
-    Age:number;
-    Gender?:string;
-    Address?:string;
-    AdmissionMonth?:string;
-    Course?:string[];
+    Id : number;
```

Comment:
Format should follow this:
````ts
Id:    number;
Name:  string;
````
Notice the spaces. 
* No space at the left side of `:`
* Then format other parts with `tab`